src/components/organisms/pupil/lesson/index.ts

```diff
@@ -6,3 +6,4 @@ export * from "./OakLessonReviewItem";
 export * from "./OakLessonReviewQuiz";
 export * from "./OakLessonTopNav";
 export * from "./OakLessonVideoTranscript";
+export * from "./printable";
```

Comment:
these don't need to be in this subfolder. Either they belong to Lesson and have Lesson in the prefix or they belong to Quiz and have quiz in the prefix and live in the quiz folder.